examples/pytorch/speech-recognition/run_speech_recognition_ctc_streaming.py

```diff
@@ -0,0 +1,651 @@
+#!/usr/bin/env python
```

Comment:
@lhoestq should we put this in `examples/research_projects/...` first and only move to an official example once everything is aligned between stream dataset objects and non-stream dataset objects on the `datasets` side? Think this was the plan no?

Reply:
Yes that would be awesome ! Thanks :)